rero_ils/modules/loans/api.py

```diff
@@ -246,24 +246,25 @@ def create(cls, data, id_=None, delete_pid=True,
             reindex=reindex, **kwargs)
         return record
 
-    def update(self, data, dbcommit=False, reindex=False):
+    def update(self, data, commit=False, dbcommit=False, reindex=False):
```

Comment:
Normally default for `commit` is True, why it is different here?

Reply:
Because of other invenio modules i.e. invenio-circulation. The default behaviour in invenio is that after an update the commit is not effective. But if `dbcommit` is True then `commit` is done.